### DISCUSSION TITLE
How do you perform date comparisons in a `where` clause?

### DISCUSSION THREAD

**Discussion by rmzg-ll** [prisma/prisma#16016] [Q&A]
If I have a column declared as `"updatedAt" TIMESTAMP(3) NOT NULL,`

How would I write the equivalent of `WHERE updatedat > NOW() - INTERVAL '30 days'` in a prisma `where: { }` construct?

**Comment by austincrim** [comment]
Hey @rmzg-ll 👋,

You can use a combination of the `gt` option and JavaScript `Date` math.

```ts
let result = await prisma.post.findMany({
  where: {
    updatedAt: { gt: new Date(new Date().valueOf() - 30 * 24 * 60 * 60 * 1000)} // subtract number of milliseconds in a month
  }
})

```

**Reply by vladi-stevanovic** [reply]
:wave: Hello @rmzg-ll , I’m going through pending GitHub Discussions to check if you need any help. Is this issue still open?

Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
